### ISSUE TITLE
Chance of precipitation decimals

### ISSUE THREAD

**Issue by twikedk** [martykan/forecastie#649]
When I look at the screen, from time to time I see this where decimals are numbering out.
I don't see it necessarily with decimals in a 'chance of' in the first place but else I would guess there is a bug to look into.

![Screenshot_2022-01-09-07-34-37](https://user-images.githubusercontent.com/24204069/148673100-81229096-6650-4245-a88c-485b098f959a.png)

And now we are at it pressure always shows x.0 but couldn't the decimal be dropped as it is with humidity?

**Comment by robinpaulson** [comment]
Yeah, I spotted the percentage precision error, have already asked the original author. It's likely a simple fix. Good spot on the redundant trailing zero on the pressure value too.

**Comment by MetaColon** [comment]
It seems this was fixed in master with the commit a7134dc in December 2021, but didn't make it into the latest release (v1.22.1 from February 2022). How come that the last release is so long ago? Would be really nice if that fix made it into the newest version on F-Droid...
